src/connector.js

```diff
@@ -331,6 +331,23 @@ function initTpl() {
 		}
 	}
 
+	if ( !qsA11yHintHTML ) {
+			if ( lang === "fr" ) {
+			qsA11yHintHTML = 
+				`<p id="sr-qs-hint" class="hidden">
+					Appuyez sur les touches de direction orientées vers le haut et vers le bas pour vous déplacer dans les suggestions de 
+					recherche. Appuyez une fois sur la touche Entrée sur une suggestion pour la sélectionner et débuter la recherche.
+				</p>`;
```

Comment:
Please put the text content of the paragraph on the same line as its tag, otherwise it outputs a space in the HTML output on the page (e.g. `<p> App...`) ↓

```suggestion
				`<p id="sr-qs-hint" class="hidden">Appuyez sur les touches de direction orientées vers le haut et vers le bas pour vous déplacer dans les suggestions de recherche. Appuyez une fois sur la touche Entrée sur une suggestion pour la sélectionner et débuter la recherche.</p>`;
```